### ISSUE TITLE
Release does not have a runWithFlask script for linux

### ISSUE THREAD

**Issue by kirypto** [kirypto/TimelineTracker#88]

**Comment by kirypto** [comment]
This is a quick one I cobbled together when deploying on my Pi

```bash
# Ensures python output is send straight to terminal without first being buffered
export PYTHONUNBUFFERED=1

# Add the Python root to the PYTHON path relative to project root
export PYTHONPATH="${PYTHONPATH}:$(pwd)/Source/Python"

# Flask environment type
#export FLASK_ENV="development"
export FLASK_ENV="production"

# Actually run the app (assuming configuration is in a config.yaml file in the project root)
python "$(pwd)/Source/Python/adapter/runners/flask/flask_app.py" ./config.yaml
```

**Comment by kirypto** [comment]
Merged, available in v0.3.0
